list_sorting.py

```diff
@@ -0,0 +1,34 @@
+#function to sort elements according to evens, odds and characters
+def list_sort(a):
+    if isinstance(a, list):
+        #list categories
+        word_cat = ['evens','odds','chars']
+        #sorted lis
+        sorted_list = {}
```

Comment:
The variable name is misleading. You are calling it `sorted_list` yet its a dictionary